ios/RNMobileDeviceManager.m

```diff
@@ -73,4 +101,68 @@ - (NSDictionary *)constantsToExport
     }
 }
 
+
+RCT_EXPORT_METHOD(isAutonomousSingleAppModeSupported: (RCTPromiseResolveBlock)resolve
+                  rejecter:(RCTPromiseRejectBlock)reject)
+{
+    [MobileDeviceManager isASAMSupported:^(BOOL reply){
```

Comment:
`isSupported` is a bit more self documenting

Reply:
Hmm...reason I used `isAutonomousSingleAppModeSupported ` is because there's another `isSupported` method and I didn't want them to be confusing? Thoughts?

Reply:
I mean the argument `reply` is less representative than the name `isSupported`

Reply:
Ah ok I getchu. Thought you were referring to method name.